CHANGES.md

```diff
@@ -133,6 +133,7 @@ profile. This started with version 0.26.0.
 - JaneStreet profile: doesn't align infix ops with open paren (#2204, @gpetiot)
 - Re-use the type let_binding from the parser instead of value_binding, improve the spacing of let-bindings regarding of having extension or comments (#2219, @gpetiot)
 - The `ocamlformat` package now only contains the binary, the library is available through the `ocamlformat-lib` package (#2230, @gpetiot)
+- Modules attributes are now formatted right after the `module` keyword, instead of at the end of the module definition (#2247, @emiletrotignon)
```

Comment:
I think in future patches where we preserve more attributes we will edit the line instead of adding a new one, I updated with something that fits what this PR does.